### DISCUSSION TITLE
How do I disable adding empty quotes ("") when formatting XML config?

### DISCUSSION THREAD

**Discussion by longtriops** [codecadwallader/codemaid#994] [Q&A]
I have a deployment variable in a web.config that does not follow XML. When cleaning up the config with Codemaid, empty quotes are added to the code and breaks the deployment variable.

Before Codemaid -> cleanup active document

![image](https://user-images.githubusercontent.com/69731313/231819679-e9804009-c0ac-43d0-a41a-83f9dbe62892.png)

After:
![image](https://user-images.githubusercontent.com/69731313/231819802-dde10810-e48e-494b-9a6e-51ed858f6529.png)

Is there a setting to prevent this?

Thank you!

**Comment by codecadwallader** [comment]
Thanks for reporting the issue.  I think this may be triggered by Visual Studio's cleanup (which CodeMaid invokes and expands upon).  If you run Edit->Advanced->Format Document do you see the same behavior?
